### PR TITLE
feat(kleros-athena-set-stake): fetch setStake events

### PR DESCRIPTION
**Pull request by ferittuncer**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ const commander = require('commander')
 
 const scrapeKlerosTokenActivation = require('./src/scrapers/kleros-token-activation')
 const scrapeArbitrablePermissionList = require('./src/scrapers/arbitrable-permission-list')
+const scrapeKlerosAthenaSetStake = require('./src/scrapers/kleros-athena-set-stake')
```
Comment (epiqueras) [inline index.js]: `scrapeKlerosLiquidSetStake` `kleros-liquid-set-stake`

We don't use the word Athena anywhere in code.

File: index.js [hunk 2]
```diff
@@ -30,6 +31,13 @@ commander
       validateETHAddress(address) && scrapeArbitrablePermissionList(address)
   )
```
Comment (epiqueras) [inline index.js]: `// Kleros Liquid Set Stake`

File: index.js [hunk 3]
```diff
@@ -30,6 +31,13 @@ commander
...
 
+commander
+  .command('kleros-athena-set-stake <address>')
```
Comment (epiqueras) [inline index.js]: `kleros-liquid-set-stake`

File: src/scrapers/kleros-athena-set-stake.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
+const fs = require('fs')
+
+const csv = require('csv')
+
+const { web3 } = require('../utils/web3')
+const KlerosLiquid = require('../assets/contracts/kleros-liquid.json')
+
+module.exports = async address => {
+  // Initialize contracts
+  const kleros = new web3.eth.Contract(KlerosLiquid.abi, address)
```
Comment (epiqueras) [inline src/scrapers/kleros-athena-set-stake.js]: `const klerosLiquid`

File: src/scrapers/kleros-athena-set-stake.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+  console.info('Contracts initialized.')
+
+  // Initialize file and rows
+  const outStream = fs.createWriteStream(
+    `./kleros-athena-set-stake-${address}.csv`
```
Comment (epiqueras) [inline src/scrapers/kleros-athena-set-stake.js]: `kleros-liquid`

File: src/scrapers/kleros-athena-set-stake.js [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+  )
+  const rows = [['address', 'subcourtID', 'stake', 'newTotalStake']]
+
+  const events = await kleros.getPastEvents('StakeSet', {
+    fromBlock: 0
+  })
+
+  console.log(events)
```
Comment (epiqueras) [inline src/scrapers/kleros-athena-set-stake.js]: This will clog the console. Let's remove it.

File: src/scrapers/kleros-athena-set-stake.js [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+  console.info(`${events.length} StakeSet event found.`)
```
Comment (epiqueras) [inline src/scrapers/kleros-athena-set-stake.js]: events

File: src/scrapers/kleros-athena-set-stake.js [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+  // const transactions = await fetchTransactions(
```
Comment (epiqueras) [inline src/scrapers/kleros-athena-set-stake.js]: Remove unused code.

File: src/scrapers/kleros-athena-set-stake.js [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+  //   'setStake(uint96,uint128)'
+  // )
+  //
+  // console.info(`${transactions.length} transactions found.`)
+
+  for (const event of events)
+    rows.push([
+      event.returnValues._address,
+      event.returnValues._subcourtID,
+      event.returnValues._stake,
```
Comment (epiqueras) [inline src/scrapers/kleros-athena-set-stake.js]: Could be nice to divide by `1e18` here.

**Comment by ferittuncer** [comment]
Could you take a look @epiqueras ? It should be OK.

**Pull request merged**